app/Dockerfile

```diff
@@ -2,10 +2,11 @@ FROM python:3.8
 ENV PYTHONUNBUFFERED 1
 
 RUN apt-get update
-RUN apt-get install ffmpeg libsm6 libxext6  -y
+RUN apt-get install ffmpeg libsm6 libxext6 emacs libhealpix-cxx-dev -y
 
```

Comment:
Please explain why emacs is included? Plead your case as to why everyone needs this and it should be in the blast base image, or remove ;) Feel free to keep in an un-commited local copy on you machine if you find it useful :-)

Reply:
I'll remove as it's not strictly necessary but helpful to install a text editor for debugging system dependencies IMO (or blast code changes that cause errors in system deps).  E.g. I added it because some python mysql thing was executing a query on fixture imports that threw errors and I needed to see what the query was

Reply:
Thanks, yeah would be good to explore better debugging stuff in the future!